### PR TITLE
fix: Fix config handling

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

**Comment by taion** [comment]
Impetus here was `get-workspaces` -> `@manypkg/get-packages`, but I went a bit farther

File: packages/build/command.js [hunk 1]
```diff
@@ -114,7 +114,9 @@ exports.handler = async ({
   '--': passthrough,
   ...options
 }) => {
-  const monoRepo = await detectMonoRepo();
```
Comment (taion) [inline packages/build/command.js]: This always returned an object, and thus a truthy value.

File: packages/cli/index.js [hunk 2]
```diff
@@ -32,6 +34,6 @@ yargs
   .command(setCmdName('intl', intl))
   .command(setCmdName('icons', svg2c))
   .command(setCmdName('fixup-workspaces', workspaces))
-  .command(format)
-  .command(lint)
+  .command(setCmdName('format', format))
+  .command(setCmdName('lint', lint))
```
Comment (taion) [inline packages/cli/index.js]: these were showing the wrong help output

File: packages/file-butler/cli.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
-const { readPackageJson } = require('@4c/cli-core/ConfigUtilities');
 const { success } = require('@4c/cli-core/ConsoleUtilities');
+const readPkgUp = require('read-pkg-up');
```
Comment (jquense) [inline packages/file-butler/cli.js]: why remove the reexport?

**Comment by taion** [comment]
idk, what it looked like was that we didn't need to have any common logic there any more, given that `detectMonoRepo` could just be replaced by checking `tool` on the output of `getPackages`

i can move stuff back into `ConfigUtilities` if you think that's better, but it also wasn't clear to me what belonged there, e.g. there was stuff that was totally unused, and the "get TS config" stuff was only used inside `ts-doctor`

i'd be happy to move it back, though; what do you think should be centralized?

**Comment by jquense** [comment]
that's fair, I'm glad to have another package do the work. The only thing is im unsure how many things outside this repo are depending on some of those exports. Maybe we leave them them for now as deprecated and i'll try and get a sense on what if anything is relying on it elsewhere?

**Comment by taion** [comment]
AFAICT nothing outside of this repo is using `cli-core/ConfigUtilities`: https://github.com/search?q=cli-core%2FConfigUtilities&type=Code

**Comment by jquense** [comment]
great then let it go

**Pull request merged**